README.md

```diff
@@ -22,7 +22,7 @@ In details, the IDs are sourced as follows:
 
 |State| ID-Source                                                                                                    | example-id                                                                 |stable|
 |-----|--------------------------------------------------------------------------------------------------------------|----------------------------------------------------------------------------|------|
-|BW| Field `DISCH` (Dienststellenschüssel) in the JSON repsonse                                                   | `BW-04154817`                                                              |✅ likely|
+|BW| WFS UUID                                                                                                     | `BW-UUID-00000a15-a965-4999-b9ad-05895eb0fad2`                             |✅ yes|
```

Comment:
I wouldn't be sure that these are actually stable since we have seen other WFS servers in which the UUID just get re-generated when an update to the data is imported.
Maybe we can use the disch when available and fall back to the uuid when we don't have the disch? What do you think?